### DISCUSSION TITLE
Custom date range selection

### DISCUSSION THREAD

**Discussion by ishah-twilio** [twilio-labs/paste#1605] [General] [answered]
We are working on designs for Data insights page where user can select canned date range or enter in custom date range.
We looked at the new date picker component and was wondering what is the guidance on using date picker contain for customer date range.

Attached is the options that we thought is appropriate usage of the date picker. 

Any feedback or guidance is appreciated. We will be testing this designs with 4 users in the 2nd week of July - Wondering if DSys has any improvements or guidance on custom range date picker pattern.

<img width="1140" alt="custom date range" src="https://user-images.githubusercontent.com/69016180/123358753-ddb18300-d520-11eb-9fd4-4f20a8ecc13d.png">

**Comment by SiTaggart** [comment, marked as answer]
Hi @ishah-twilio 

From an accessibility, depending on the full interaction, Option 1 is the only feasible option. A combobox cannot show other or further options like it does in Option 2, I'm afraid. It can't act like a menu, which does support further or sub options, because there is no current way to communicate that to assistive technology.

Maybe in the future when the specifications are more flexible we'll be able to do things like in option 2, but for now the best option is to keep it very simple like in Option 1, with 3 separate dedicated inputs for each field.
